environment.yml

```diff
@@ -0,0 +1,11 @@
+name: claymodel
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - conda-lock~=2.4.2
+  - jupyterlab~=4.0.7
+  - lightning~=2.1.0
+  - planetary-computer~=1.0.0
+  - python=3.11
+  - stackstac~=0.5.0
```

Comment:
Have kept this dependency list fairly minimal for now, but I've tested it to work for the Pull Request at #11. Happy to add more dependencies to this list on request (or can be done in follow-up PRs).